core/api/service/author/impl/author_api_impl.cpp

```diff
@@ -64,8 +64,10 @@ namespace kagome::api {
       const gsl::span<const uint8_t> &seed,
       const gsl::span<const uint8_t> &public_key) {
     if (not(crypto::KEY_TYPE_BABE == key_type)
-        && not(crypto::KEY_TYPE_GRAN == key_type)) {
-      SL_INFO(logger_, "Unsupported key type, only BABE and GRAN are accepted");
+        && not(crypto::KEY_TYPE_GRAN == key_type)
```

Comment:
Seems like it might be good idea to extract these into an array and search it here. And also use it for the log below.